tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -4,26 +4,33 @@
 import math
 import os
 import pathlib
-import shutil
-from argparse import Namespace
+import sys
 from typing import Callable, Optional, cast
 from unittest.mock import ANY, MagicMock, patch
 
-import pytest
-import torch
-import transformers
 from composer import Trainer
 from composer.loggers import MLFlowLogger
 from composer.utils import dist, get_device
+
+from llmfoundry.callbacks import HuggingFaceCheckpointer
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+
+# Add repo root to path so we can import scripts and test it
```

Comment:
How come this wasn't handled by the fixtures? / Why were these tests passing in CI?